File: MonitoreoCNET/MonitoreoCNET/Models/Persona/Interes.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using MonitoreoCNET.Models.Organizaciones;
using MonitoreoCNET.Models.Servicios;

namespace MonitoreoCNET.Models.Persona
{
    [Table("interes")]
    public class Interes
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        [ForeignKey("id_entidad")]
        public virtual Entidad Entidad { get; set; }

        [ForeignKey("id_servicio")]
        public virtual Servicio Servicio { get; set; }

        [ForeignKey("id_miembro")]
        public virtual Miembro Miembro { get; set; }

        public Interes()
        {
        }
    }
}

File: monitoreo/src/main/java/models/entidades/persona/Interes.java
package models.entidades.persona;

import models.entidades.organizaciones.entidad.Entidad;
import models.entidades.servicios.servicio.Servicio;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name="interes")
public class Interes {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Getter @Setter
    private int id;

    @Getter @Setter
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "id_entidad", referencedColumnName = "id")
    private Entidad entidad;

    @Getter @Setter
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "id_servicio", referencedColumnName = "id")
    private Servicio servicio;

    @Getter @Setter
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "id_miembro",referencedColumnName = "id")
    private Miembro miembro;

    public Interes() {
    }
}
